test/recipes/30-test_evp_data/evppkey_ffdhe.txt

```diff
@@ -105,7 +105,7 @@ CEKAlg=id-aes128-wrap
 Ctrl = dh_pad:1
 SharedSecret=89A249DF4EE9033B89C2B4E52072A736D94F51143A1ED5C8F1E91FCBEBE09654
 
-# FIPS 3.0.0 has a bug in the default padding #17859
+# FIPS 3.0.0 has a bug which prevents padding to be set #17859
```

Comment:
probably still not quite right :) 3.0.0 allows the padding to be set. Now it ignores the set..

Reply:
Grrrr.....